lib/conf/tasks/daemon.js

```diff
@@ -135,16 +137,40 @@ exports.remove = function(cb) {
 }
 
 exports.set_watcher = function(cb) {
-  satan.status(watcher_opts.key, function(err, running) {
-    if (running) console.log("Prey monitor it's already running, restarting...")
+  exec(`${join(paths.current, 'bin', 'prey-user')} -v`, (error, prey_user_version) => {
+    if (error) return;
+    logger.info(`prey-user output: ${prey_user_version}`);
+    queryStorage(prey_user_version, cb);
+  });
+}
 
-    satan.ensure_created(watcher_opts, function(err) {
-      if (err) return cb(err);
+const queryStorage = (preyUserVersion, cb) => {
+  storage.do('query', {type: 'keys', column: 'id', data: 'prey_user_version'}, (err, stored_data) => {
+    if (err || stored_data && stored_data.length == 0) {
```

Comment:
should parentheses be added?

Reply:
Since (err || stored_data && stored_data.length == 0) is the same as (err || (stored_data && stored_data.length == 0)) it doesn't matter tbh.